<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1900</width>
    <height>1180</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Rolechoose_Diana">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>1010</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Diana</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Rolechoose_Ava">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>80</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ava</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Rolechoose_Bella">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>360</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bella</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Rolechoose_Eileen">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>650</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eileen</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>540</y>
     <width>302</width>
     <height>246</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="newgamebutton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Aa可爱の日系中文2万字</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadbutton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Aa可爱の日系中文2万字</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>1000</y>
     <width>800</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1900</width>
     <height>1180</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Aa可爱の日系中文2万字</family>
     <pointsize>13</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>关注Asoul谢谢喵！
Asoul的四位小姐姐：
嘉然，向晚，乃琳，贝拉都超级可爱！</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
